tools/cli/internal/cli/changelog/create_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package changelog
+
+import (
+	"testing"
+
+	"github.com/mongodb/openapi/tools/cli/internal/cli/flag"
+	"github.com/mongodb/openapi/tools/cli/internal/test"
+)
+
+func TestCreateBuilder(t *testing.T) {
```

Comment:
[q] will we add more unit tests for create.go in another ticket?

Reply:
no, the command is only calling changelog functions and saving files. We already have unit tests for changelog and we are going also to add e2e tests so I think are okay. Let me know if you were thinking on specific unit tests scenarios that we could add for the create command and I will add them in a follow up PR

Reply:
yeah, I'd like to test the saving files logic for sure since we can find bugs also in the way we write files/folders etc - great if it can be a follow-up!